core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -135,16 +136,22 @@ private static CountingInputStream openStream(Playable playable, Context context
         }
     }
 
-    public static List<Chapter> loadChaptersFromUrl(String url) {
+    public static List<Chapter> loadChaptersFromUrl(String url, boolean refreshed) {
```

Comment:
Please rename to `forceRefresh` or something similar (same with the other method here). Just the word `refreshed` does not really tell what it stands for.